process-services/1.11/config/database.md

```diff
@@ -0,0 +1,82 @@
+|datasource.password|The password of the above user.|
+
+Example:
+
+```text
+datasource.driver=com.mysql.jdbc.Driver
+datasource.url=jdbc:mysql://127.0.0.1:3306/activiti?characterEncoding=UTF-8
+
+datasource.username=alfresco
+datasource.password=alfresco
+```
+
+### Connection pooling
+
+When using JDBC Connection Parameters, you can configure the following connection pool settings 
+to suit the anticipated load:
+
+|Property|Description|
+|--------|-----------|
+|datasource.min-pool-size|The minimum number of connections in the connection pool. For example `5`. |
+|datasource.max-pool-size|The maximum number of connections in the connection pool. For example `100`. |
+|datasource.acquire-increment|The number of additional connections the system will try to acquire each time the connection pool is exhausted. For example `5`. |
+|datasource.preferred-test-query|The query used to verify that the connection is still valid. The value depends on the database: `select 1` for H2, MySQL, PostgreSQL and Microsoft SQL Server, `SELECT 1 FROM DUAL` for Oracle and `SELECT current date FROM sysibm.sysdummy1` for DB2.|
+|datasource.test-connection-on-checkin|Boolean value. If `true`, an operation will be performed asynchronously on every connection checkin to verify that the connection is valid. For best performance, a proper `datasource.preferred-test-query` should be set.|
+|datasource.test-connection-on-checkout|Boolean value. If `true`, an operation will be performed asynchronously on every connection checkout to verify that the connection is valid. Testing Connections on checkout is the simplest and most reliable form of Connection testing. For best performance, a proper `datasource.preferred-test-query` should be set.|
+|datasource.max-idle-time|The number of seconds a connection can be pooled before being discarded. For example `1800`. |
+|datasource.max-idle-time-excess-connections|Number of seconds that connections in excess of `minPoolSize` should be permitted to remain idle in the pool before being discarded. The intention is that connections remain in the pool during a load spike. For example `1800`. |
+
+The connection pooling framework used is [C3P0](http://www.mchange.com/projects/c3p0/){:target="_blank"}. It has extensive documentation on the settings described above.
+
+## JNDI data source
+
+If a JNDI data source is configured in the web container or application server, the JNDI name should be set with the following properties:
+
+|Property|Description|
+|--------|-----------|
+|datasource.jndi.name|The JNDI name of the datasource. This varies depending on the application server or web container. For example `jdbc/activitiDS`. |
+|datasource.jndi.resourceRef|Set whether the look up occurs in a J2EE container, that is, if the prefix `java:comp/env/` needs to be added if the JNDI name doesn’t already contain it. For example `true`. |
+
+Example (on JBoss EAP 6.3):
```

Comment:
JBoss isn't supported in both APS 1.x and 2.x. We need to review JNDI data source usage.

Reply:
Chatted with Bassam will complete this part after docs release